docs/src/pages/components/Icon.vue

```diff
@@ -47,6 +47,12 @@
                 <md-table-cell><code>String</code></md-table-cell>
                 <md-table-cell>The src of the image file. Can be SVG icons too!</md-table-cell>
               </md-table-row>
+              <md-table-row>
+                <md-table-cell>md-iconset</md-table-cell>
+                <md-table-cell><code>String</code></md-table-cell>
+                <md-table-cell>The icon set font you'll use. For example, in 
```

Comment:
I think that `md-iconset` should be a String prop. To be used like that:
``` vue
<md-icon md-iconset="fa fa-home"></md-icon> // In case of FA for example
```
``` vue
<md-icon>home</md-icon> // In case of Material Icons
```